numpy/distutils/mingw32ccompiler.py

```diff
@@ -390,6 +391,13 @@ def _build_import_library_amd64():
                   (out_file))
         return
 
+    # didn't exist in virtualenv, maybe in base distribution?
+    base_file = os.path.join(sys.base_prefix, 'libs', out_name)
+    if os.path.isfile(base_file):
+        log.debug('Skip building import library: "%s" exists' %
+                  (base_file))
```

Comment:
If you intend a tuple here, need a `,`, otherwise don't see the need for the parenthesis.